test/PostageStamp.test.ts

```diff
@@ -1238,30 +1238,30 @@ describe('PostageStamp', function () {
         expect(stamp[4]).to.equal(11);
       });
 
-      it('should transfer the token', async function () {
-        await postageStampStamper.copyBatch(
-          stamper,
-          batch.initialPaymentPerChunk,
-          batch.depth,
-          batch.bucketDepth,
-          batch.nonce,
-          batch.immutable
-        );
-        expect(await token.balanceOf(stamper)).to.equal(0);
-      });
-
-      it('should not create batch if insufficient funds', async function () {
-        await expect(
-          postageStampStamper.copyBatch(
-            stamper,
-            batch.initialPaymentPerChunk + 1,
-            batch.depth,
-            batch.bucketDepth,
-            batch.nonce,
-            batch.immutable
-          )
-        ).to.be.revertedWith(errors.erc20.exceedsBalance);
-      });
+      // it('should transfer the token', async function () {
+      //   await postageStampStamper.copyBatch(
+      //     stamper,
+      //     batch.initialPaymentPerChunk,
+      //     batch.depth,
+      //     batch.bucketDepth,
+      //     batch.nonce,
+      //     batch.immutable
+      //   );
+      //   expect(await token.balanceOf(stamper)).to.equal(0);
+      // });
+
+      // it('should not create batch if insufficient funds', async function () {
+      //   await expect(
+      //     postageStampStamper.copyBatch(
+      //       stamper,
+      //       batch.initialPaymentPerChunk + 1,
+      //       batch.depth,
+      //       batch.bucketDepth,
+      //       batch.nonce,
+      //       batch.immutable
+      //     )
+      //   ).to.be.revertedWith(errors.erc20.exceedsBalance);
+      // });
```

Comment:
?

Reply:
Because we are removing erc20 transfers, if you remember we talked about withdrawing funds, if we withdraw then we just fund the new contract with that amount we dont transfer erc20 with each trx and there is really no need to as we dont track amount of funds each person has, its all pooled together.